search/src/components/ScoreChunk.tsx

```diff
@@ -337,12 +332,7 @@ const ScoreChunk = (props: ScoreChunkProps) => {
                 }
                 tooltipText="Copy to clipboard"
               />
-              <Show
-                when={
-                  props.setOnDelete &&
-                  $currentUser()?.id == props.chunk.author?.id
-                }
-              >
+              <Show when={props.setOnDelete}>
```

Comment:
Can we grab the user role to check if they can delete?

Reply:
Yes